masschallenge/django-accelerator
# AC-8354: Update references to "BulletTrain" in the codebase to reflect company name change

Changes introduced in [AC-8354](https://masschallenge.atlassian.net/browse/AC-8354):
- Removed references to Bullet train.

How to test:
- Check that all the references to variants on the phrase "bullet train" are updated to refer to "flagsmith" instead.
- bullet-train SDKs (accelerate and front-end) are replaced by flagsmith clients 

Note:
In the flagsmith python client, the package wasn't updated as well as the setup instructions
https://github.com/Flagsmith/flagsmith-python-client

Sibling PRs:
[Frontend](https://github.com/masschallenge/front-end/pull/275)
[Accelerate](https://github.com/masschallenge/accelerate/pull/2875)
